src/PostgREST/Unix.hs

```diff
@@ -38,3 +44,29 @@ runAppInSocket settings app socketFileMode sockPath = do
   runSettingsSocket settings sock app
   -- clean socket up when done
   close sock
+
+
+-- SIGNALS
+
+data SignalHandlers = SignalHandlers
+  { fullConfigReReader :: IO ()
+  , connWorker         :: IO ()
+  , releasePool        :: IO ()
+  }
```

Comment:
I don't think this record really improves the interface of `installSignalHandlers`. Why not just 3 regular arguments?